src/main/java/org/broadinstitute/hellbender/utils/gcs/BucketUtils.java

```diff
@@ -388,6 +389,22 @@ public static String getPathWithoutBucket(String path) {
         return CloudStorageFileSystem.forBucket(BUCKET).getPath(pathWithoutBucket);
     }
 
+    /**
+     * String -> Path. This will accept either
+     * "folder/file", or
+     * "file://folder/file", or
+     * "gs://bucket/folder/file".
+     * It will only accept "hdfs://" if you have an NIO filesystem provider installed for HDFS.
+     */
+    public static java.nio.file.Path getPath(String fileNameOrUrl) {
```

Comment:
Sounds good. That one can throw IOException. Are we OK with this pattern?

```java
// some function that doesn't throw IOException
try {
  // code goes here, BucketUtils -> IOUtils
} catch (IOException io) {
  throw new RuntimeException(io);
}
```
?